Makefile

```diff
@@ -1,54 +1,58 @@
-# Declare subcommands as "phony" targets, since they're not directories.
```

Comment:
Perhaps help should become default action for `make`? For now, it appears that `make` defaults to `make clean`  (by setting `DEFAULT_GOAL=help` here ?)

Reply:
Great idea, @emkll, will set the default goal to the help target!